src/tokens/tokens.ts

```diff
@@ -0,0 +1,59 @@
+import {legacyTokens} from './_legacy-tokens';
+import {lightColorScheme, darkColorScheme} from './_color';
+import {motion} from './_motion';
+import {typography} from './_typography';
+
+/**
+ * Values to convert to CSS custom properties.
+ * @example {background: '#000'} // --p-background: #000;
+ */
+export interface TokenObject {
+  [token: string]: string;
+}
```

Comment:
**suggestion:** I was actually thinking about renaming this in my original PR, but didn't want to change anything after all the checks were passing. What do you think about naming this `TokenGroup` or `TokensGroup`. That's effectively how I was architecting the tokens structure (See the [Group section of the design tokens format module](https://design-tokens.github.io/community-group/format/#groups-0) for more information).

Notice I use this convention in the [getStaticCustomProperties](https://github.com/Shopify/polaris-react/blob/60cc0a2bf8f00219a200d19d3d418e58a1668145/src/components/CustomProperties/styles.ts#L56) utility

I also subjectively prefer this over tacking on the literal `Object` type 🙂